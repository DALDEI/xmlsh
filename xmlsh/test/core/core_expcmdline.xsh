# Test expanding of command line args
. ../common
C=echo
$C loc() 
C="echo foo"
$C loc()
echo loc() next line should fail
"$C" loc()
echo loc() shold catch failure
"$C" loc || echo Success 
