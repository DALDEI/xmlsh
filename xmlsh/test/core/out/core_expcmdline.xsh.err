[common.xsh line: 13]
"$C": not found
[core_expcmdline.xsh line: 10]
"$C": not found
