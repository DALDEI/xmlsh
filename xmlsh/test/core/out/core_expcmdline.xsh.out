[4]
foo [6]
[7] next line should fail
[9] shold catch failure
Success
